scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala

```diff
@@ -2,42 +2,8 @@ package com.example.helloworld
 
 import scalapb.zio_grpc.{ServerMain, ServiceList}
 
-import java.util.concurrent.Executors
-
 object GreeterServer extends ServerMain {
-  def services = ServiceList.add(GreeterImpl)
-
-  override def builder = {
-    val sb = super.builder
-
-    /**
-     * Allow customization of the Executor with two environment variables:
-     *
-     * <p>
-     * <ul>
-     * <li>JVM_EXECUTOR_TYPE: direct, workStealing, single, fixed, cached</li>
-     * <li>JVM_EXECUTOR_THREADS: integer value.</li>
-     * </ul>
-     * </p>
-     *
-     * The number of Executor Threads will default to the number of
-     * availableProcessors(). Only the workStealing and fixed executors will use
-     * this value.
-     */
-    val threads = System.getenv("JVM_EXECUTOR_THREADS")
-    var i_threads = Runtime.getRuntime.availableProcessors
-    if (threads != null && threads.nonEmpty) i_threads = threads.toInt
-
-    val value = System.getenv.getOrDefault("JVM_EXECUTOR_TYPE", "workStealing")
-    value match {
-      case "direct" => sb.directExecutor
-      case "single" => sb.executor(Executors.newSingleThreadExecutor)
-      case "fixed" => sb.executor(Executors.newFixedThreadPool(i_threads))
-      case "workStealing" => sb.executor(Executors.newWorkStealingPool(i_threads))
-      case "cached" => sb.executor(Executors.newCachedThreadPool)
-    }
-    sb
```

Comment:
Why did you remove this logic?

Reply:
That is surprising they wouldn't add an alternative approach. Could you please point me to a relevant changelog entry?

Reply:
It's hard to get details from public information.

In my opinion, `zio-grpc` api is not stable, the authors changed it many times. And this builder configuration will not have a great impact on the test results.

Reply:
Alright, thanks.